pootle/apps/pootle_app/management/commands/config.py

```diff
@@ -0,0 +1,224 @@
+
+from pootle.core.delegate import config
+
+
+class Command(BaseCommand):
+    help = "Manage configuration for Pootle objects."
+
+    def add_arguments(self, parser):
+        super(Command, self).add_arguments(parser)
+        parser.add_argument(
+            'content_type',
+            action='store',
+            nargs="?",
+            help=_("Get/set config for given content type"))
+        parser.add_argument(
+            'object',
+            action='store',
+            nargs="?",
+            help=_("Unique identifier for object, by default pk"))
+        parser.add_argument(
+            '-o', '--object-field',
+            action='store',
+            dest='object_field',
+            help=_(
+                "Use given field as identifier to get object, "
+                "should be unique"))
+        group = parser.add_mutually_exclusive_group()
+        group.add_argument(
+            '-g', '--get',
+            action='store',
+            dest='get',
+            metavar="KEY",
+            help=_("Get config with this key, key must be unique"))
+        group.add_argument(
+            '-l', '--list',
+            action='append',
+            dest='list',
+            metavar="KEY",
+            help=_(
+                "List config with this key, "
+                "can be specified multiple times"))
+        group.add_argument(
+            '-s', '--set',
+            action='store',
+            dest="set",
+            metavar=("KEY", "VALUE"),
+            nargs=2,
+            help=_(
+                "Set config with this VALUE, KEY should not exist "
+                "or must be unique"))
+        group.add_argument(
+            '-a', '--append',
+            action='store',
+            dest="append",
+            metavar=("KEY", "VALUE"),
+            nargs=2,
+            help=_("Append config with this value"))
+        group.add_argument(
+            '-c', '--clear',
+            action='append',
+            dest='clear',
+            metavar="KEY",
+            help=_("Clear config with this key"))
+        parser.add_argument(
+            '-j', '--json',
+            action='store_true',
+            dest='json',
+            help=_(
+                "When getting/setting/appending a config dump/load "
+                "VALUE as json"))
+
+    def print_config_item(self, item, object_field=None):
+        if item.content_type:
+            ct = ".".join(item.content_type.natural_key())
+            if item.object_pk:
+                if object_field:
+                    pk = getattr(
+                        item.content_object,
+                        object_field)
+                else:
+                    pk = item.object_pk
+                item_name = "%s[%s]" % (ct, pk)
+            else:
+                item_name = ct
+        else:
+            item_name = "Pootle"
+        self.stdout.write(
+            "{: <25} {: <25} {: <30}".format(
```

Comment:
column formatted output, the cols could probs do with being made wider, perhaps dynamically